runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/WindmillStateCacheTest.java

```diff
@@ -130,7 +133,8 @@ private static StateNamespace triggerNamespace(long start, int triggerIdx) {
 
   @Before
   public void setUp() {
-    cache = new WindmillStateCache();
+    options = PipelineOptionsFactory.as(DataflowWorkerHarnessOptions.class);
+    cache = new WindmillStateCache(options.getWorkerCacheMb());
     assertEquals(0, cache.getWeight());
```

Comment:
Can you add a check to this test to make sure that the `maximumWeight` of the cache is the 100 MB? (perhaps use a number different than 100 to be sure).

Reply:
Fixed this by adding a new Test method in WindmillStateCacheTest class. I created a new getter in the WindmillStateCache to retrieve the size of max weight on bytes, and compared it to the initial value set